### PR TITLE
[fix](heartbeat) fix heartbeat editlog no persist hbTime

### PR DESCRIPTION
**Pull request by yujun777**
Backend persist lastUpdateMs,  it will be modified by heartbeat editlog. But heartbeat editlog not persist hbTime,   and hbTime always equal 0,   it will make  backend's lastUpdateMs = 0 in bdb image.

fix details:
1. heartbeat response persist hbTime;
2. only be state change will write an editlog.  but we make a change:  even a backend is healthy, still write a healthy response editlog every 5 min. Inorder to make backend's lastUpdateMs periodly updated in bdb image. But notice that this change wouldn't increase real editlog num. Because heartbeat mgr will patch all fe/be's  heartbeat into one editlog.  Even no fe/be state change, it still write an editlog which not contains any node's response.
3. for a dead heartbeat response,  set hbTime to last succ hbTime,  then replayer can set correct lastUpdateMs;

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run feut

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run feut

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by yujun777** [comment]
run buildall

**Review by deardeng — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/system/Backend.java [hunk 1]
```diff
@@ -898,6 +912,13 @@ public boolean handleHbResponse(BackendHbResponse hbResponse, boolean isReplay)
             lastMissingHeartbeatTime = System.currentTimeMillis();
         }
 
+        if (isChanged) {
+            Random random = new SecureRandom();
+            int second = random.nextInt(Math.max(Config.heartbeat_interval_second,
+                    Config.editlog_healthy_heartbeat_seconds));
+            this.lastEditlogHeartbeatTime = System.currentTimeMillis() + second * 1000L / 2;
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/system/Backend.java:919]: lastEditlogHeartbeatTime is a future time?

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run performance

**Comment by yujun777** [comment]
run performance

**Comment by yujun777** [comment]
run performance

**Comment by yujun777** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Comment by yujun777** [comment]
run performance

**Pull request merged**